### PR TITLE
Wdfn 485

### PR DESCRIPTION
**Pull request by mbucknell**
Please note that the sprite icons in the footer will not appear when using a separate static server as is done typically in development.

### CODE REVIEW

File: wdfn-server/waterdata/templates/partials/footer.html [hunk 1]
```diff
@@ -6,6 +6,40 @@
             <a class="usgs-footer-link" href="https://www2.usgs.gov/laws/accessibility.html">Accessibility</a> |
             <a class="usgs-footer-link" href="https://www.usgs.gov/sitemap">Site Map</a> |
             <a class="usgs-footer-link" href="https://answers.usgs.gov/">Contact USGS</a>
+
+            <div class="social-media-links">
+                <span>Follow</span>
+                <a href="https://twitter.com/usgs" target="_blank">
+                    <svg class="usa-icon usa-icon--size-3" aria-label="USGS twitter" role="img">
+                        <use xlink:href="{{ 'img/sprite.svg#twitter' | asset_url }}"></use>
```
Comment (abriggs-usgs) [inline wdfn-server/waterdata/templates/partials/footer.html:14]: Not sure if this is an issue or not, but I bumped into at some point in the past. The 'xlink:href' is listed as depreciated: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href.

**Review by abriggs-usgs — APPROVED** [review]
Looks like a lot of work for adding icons ;)

File: wdfn-server/waterdata/templates/partials/footer.html (hunk 1 above)
Comment (mbucknell) [reply wdfn-server/waterdata/templates/partials/footer.html:14]: I asked this question on the uswds-public Slack channel. If you don't use that, it won't work on Safari, so it's not really deprecated.

Comment (abriggs-usgs) [reply wdfn-server/waterdata/templates/partials/footer.html:14]: Well that is kinda dumb, but good to know.

Comment (jkreft-usgs) [reply wdfn-server/waterdata/templates/partials/footer.html:14]: Safari, the new IE

**Pull request merged**